backend/apps/models/user.py

```
@@ -0,0 +1 @@
+e
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)